### PR TITLE
[Outlook] (recurrence) Shows how to get recurrence and seriesId

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: samples/outlook/50-recurrence/get-recurrence-appointment-organizer.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        $("#get-recurrence").click(getRecurrence);
+
+        function getRecurrence() {
+          Office.context.mailbox.item.recurrence.getAsync(function(asyncResult) {
+            if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+              var recurrence = asyncResult.value;
+              if (recurrence === null) {
+                console.log("This is a single appointment.");
+              } else {
+                console.log("Recurrence pattern: " + JSON.stringify(recurrence));
```
Comment (Oleg-O) [inline samples/outlook/50-recurrence/get-recurrence-appointment-organizer.yaml]: I found that just logging objects will result in ScriptLab UI that allows you to drill into JSON, which may be better than dumping JSON.stringify result?...

```suggestion
                console.log(recurrence);
```

File: samples/outlook/85-tokens-and-service-calls/basic-rest-cors.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 order: 5
 id: outlook-tokens-and-service-calls-basic-rest-cors
 name: Get a message using REST
-description: 'Gets a message using REST without any back-end code.'
```
Comment (Oleg-O) [inline samples/outlook/85-tokens-and-service-calls/basic-rest-cors.yaml]: why does this keep on changing :-)  are we using different versions of Yarn or something? I installed latest very recently...

File: samples/outlook/50-recurrence/get-series-id.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
+order: 1
+id: outlook-recurrence-get-series-id
+name: Get seriesId
```
Comment (Oleg-O) [inline samples/outlook/50-recurrence/get-series-id.yaml]:
```suggestion
name: Get the series ID
```

File: samples/outlook/50-recurrence/get-recurrence-read.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+        function getRecurrence() {
+          var recurrence = Office.context.mailbox.item.recurrence;
+
+          if (recurrence === undefined) {
+            console.log("This item is a message but not a meeting request.");
+          } else if (recurrence === null) {
+            console.log("This is a single appointment.");
+          } else {
+            console.log("Recurrence pattern: " + JSON.stringify(recurrence));
```
Comment (Oleg-O) [inline samples/outlook/50-recurrence/get-recurrence-read.yaml]:
```suggestion
                console.log(recurrence);
```

File: samples/outlook/85-tokens-and-service-calls/basic-rest-cors.yaml (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply samples/outlook/85-tokens-and-service-calls/basic-rest-cors.yaml]: not sure. I'm using 1.12.3

**Comment by ElizabethSamuel-MSFT** [comment]
@Oleg-O For now, I'm doing the get for each object. Sometime soon, I hope to include add/update/delete/remove etc where applicable.

File: samples/outlook/85-tokens-and-service-calls/basic-rest-cors.yaml (hunk 2 above)
Comment (Oleg-O) [reply samples/outlook/85-tokens-and-service-calls/basic-rest-cors.yaml]: 1.19.1 here

**Pull request merged**